plugins/async-node/components/package.json

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "@player-ui/async-node-plugin-components",
+  "version": "0.0.0-PLACEHOLDER",
+  "main": "src/index.tsx",
```

Comment:
I think the build is failing b/c this doesn't actually point to anything

Reply:
Yeah you shouldn't need to manually define this, it gets automatically added when the `package.json` is put together in the Bazel rules

Reply:
There would need to be a root `index.tsx` file that exports the `async.tsx` file.

Reply:
well I have taken reference-assets as plugin and it doesn't have a separate index file as mentioned its just that instead of index.tsx , I have named it as async.tsx 
![image](https://github.com/player-ui/player/assets/171299777/ebcb0b86-07c3-4adf-9857-bfafdb029bb4)
Please can you suggest , ty